LICENSE

```diff
@@ -199,68 +199,3 @@
    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
```

Comment:
Let's also create a NOTICE file, here is an example https://www.apache.org/licenses/example-NOTICE.txt

If we have required third-party notices or licenses, we shall also record those portions

Reply:
Sure, Thank you, @yaooqinn .